### PR TITLE
Updates to utilize dyno id including task id for matching to ranch

### PR DESCRIPTION
**Pull request by swettk**

### CODE REVIEW

File: bin/start [hunk 1]
```diff
@@ -35,6 +35,12 @@ export MEMORY_AVAILABLE=$MEMORY_AVAILABLE
 export WEB_MEMORY=$WEB_MEMORY
 export WEB_CONCURRENCY=$WEB_CONCURRENCY
 
-export DYNO="${PROCESS:-run}.$(hostname)"
+if [ -f "${ECS_CONTAINER_METADATA_FILE:-}" ]
+then
+  export TASKID=$(cat ${ECS_CONTAINER_METADATA_FILE} | jq .TaskARN | awk '{split($0,a,"-"); print a[7]}' | sed 's/\"//g')
```
Comment (bobzoller) [inline bin/start]: `jq -r` and you can kill the last pipe to `sed` :+1:

File: bin/start [hunk 2]
```diff
@@ -35,6 +35,12 @@ export MEMORY_AVAILABLE=$MEMORY_AVAILABLE
 export WEB_MEMORY=$WEB_MEMORY
 export WEB_CONCURRENCY=$WEB_CONCURRENCY
 
-export DYNO="${PROCESS:-run}.$(hostname)"
+if [ -f "${ECS_CONTAINER_METADATA_FILE:-}" ]
```
Comment (bobzoller) [inline bin/start]: is the `:-` extraneous here?

File: bin/start [hunk 3]
```diff
@@ -35,6 +35,12 @@ export MEMORY_AVAILABLE=$MEMORY_AVAILABLE
...
+then
+  export TASKID=$(cat ${ECS_CONTAINER_METADATA_FILE} | jq .TaskARN | awk '{split($0,a,"-"); print a[7]}' | sed 's/\"//g')
+  export DYNO="${PROCESS:-run}.${TASKID}"
```
Comment (bobzoller) [inline bin/start]: DRY up the `${PROCESS:-run}` bit

**Review by bobzoller — CHANGES_REQUESTED** [review]

File: bin/start (hunk 2 above)
Comment (swettk) [reply bin/start]: Nope, required

File: bin/start (hunk 3 above)
Comment (swettk) [reply bin/start]: not sure what you're thinking there....

File: bin/start (hunk 2 above)
Comment (bobzoller) [reply bin/start]: because of bash `set -o nounset` above! got it

**Review by bobzoller — APPROVED** [review]

**Pull request merged**